pages/tools/starkli.mdx

```diff
@@ -0,0 +1,61 @@
+import { Steps } from 'nextra-theme-docs'
+
+# Starkli Overview
+[Starkli](https://www.starknetjs.com/) is a lightweight JavaScript/TypeScript library enabling interaction between your DApp and Starknet Inspired by ethers.js, it offers a familiar API and allows you to interact with Starknet accounts, providers, and contracts. Key functionalities include account management, network interaction, contract interfacing, and deployment automation.
+
+## Installing Starkli
+<Steps>
+### Installing starkliup
+Install Starkliup, the installer for the Starkli environment
+```bash
+curl https://get.starkli.sh | sh
+```
+Starkliup should now be installed.
+Restart the terminal
+
+### Install starkli v0.1.20
+Run starkliup to install starkli v0.1.20
```

Comment:
specify that Madara is compatible with 0.1.20 right now but will very soon be compatible with the latest version